parser/src/lib.rs

```diff
@@ -30,10 +30,18 @@ pub struct Parameter {
 /// All display hints
 #[derive(Clone, Debug, PartialEq)]
 pub enum DisplayHint {
+    NoHint {
+        zero_pad: Option<ZeroPad>,
```

Comment:
I think these `zero_pad` fields could just be of type `usize` and fall back to 0 if no zero-padding is requested. That should simplify the `format_args` code.

Reply:
Ok converted to using a flat usize, with `0` being equivalent to `None`.